UI/ClientUI.h

```diff
@@ -51,6 +52,8 @@ class ClientUI {
     ShipDesignManager* GetShipDesignManager() { return m_ship_designs.get(); };
 
     void    GetSaveGameUIData(SaveGameUIData& data) const;              //!< populates the relevant UI state that should be restored after a save-and-load cycle
+
+    std::string FormatTimestamp(boost::posix_time::ptime timestamp) const;
```

Comment:
document with comment... purpose is unclear

does this need to be a member function?

Reply:
I suppose I can make it static function.

Reply:
Fixed in base PR.